doc/upgrading.md

```diff
@@ -24,7 +24,8 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 
 #### [RMB-862](https://issues.folio.org/browse/RMB-862) Upgrade to Vert.x 4.1.2
```

Comment:
Shouldn't you mentions RMB-874 (this one) as well?

Reply:
Yes, added, thanks!